apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx

```diff
@@ -1143,8 +1143,8 @@ export const PoolPoster = ({
                       {intl.formatMessage(
                         {
                           defaultMessage:
-                            "Please contact the <a>{name} team</a> by email if you have <strong>any questions</strong> or <strong>require an accommodation</strong> during this process.",
-                          id: "rKUVdL",
+                            "Please <a>{name}contact the team</a> by email if you have <strong>any questions</strong> or <strong>require an accommodation</strong> during this process.",
```

Comment:
What's up with the {name} variable?  I guess it's always blank?  It's going to look very awkward if it ever populates.  It seems like it's supposed to be removed in the copy provided in this issue.

Reply:
Oh you're right. I just assumed it was there for a reason like accessibility text but that isn't the case at all here
https://github.com/GCTC-NTGC/gc-digital-talent/pull/11706/commits/58bb3362dd3ede70d1ab21e3a756a926910c41e3